### PR TITLE
app-admin/logstash-bin: bump to 5.6.13/6.4.3

### PR DESCRIPTION
**Pull request by hydrapolic**

### CODE REVIEW

File: app-admin/logstash-bin/logstash-bin-5.6.13.ebuild [hunk 1]
```diff
@@ -0,0 +1,77 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```
Comment (juippis) [inline app-admin/logstash-bin/logstash-bin-5.6.13.ebuild]: Can 7 be used in here? (Same applies to 2nd version bump)

Comment (hydrapolic) [reply app-admin/logstash-bin/logstash-bin-5.6.13.ebuild]: Unfortunately, the inherited java-utils-2 eclasses uses versionator, which is banned in EAPI=7.

Comment (juippis) [reply app-admin/logstash-bin/logstash-bin-5.6.13.ebuild]: Yep, saw it got inherited and had a feeling there might be something. Thanks for checking!

Comment (hydrapolic) [reply app-admin/logstash-bin/logstash-bin-5.6.13.ebuild]: Thanks for the review :)

**Comment by hydrapolic** [comment]
Thanks @Whissi

**Pull request closed without merging**